package.json

```diff
@@ -47,6 +47,7 @@
     "expo-sqlite": "~10.2.0",
     "expo-web-browser": "~10.2.1",
     "flow-enums-runtime": "^0.0.6",
+    "hermes-parser": ">=0.4.8",
```

Comment:
This happens in the commit whose message is just "deps: Run `yarn upgrade`". I reproduce that `yarn upgrade` causes this line to get added. Odd.

Presumably related to the hermes-parser line in our `resolutions` section. Perhaps drop that? At this point our dependencies that pull it in all depend on newer versions than this anyway.